test/ctia/bundle/routes_test.clj

```diff
@@ -133,10 +134,11 @@
                                   external_id)
                           :headers {"Authorization" "45c1f5e3f05d0"})
             [entity :as entities] (:parsed-body response)]
-        (is (= 1 (count entities))
-            "Only one entity is linked to the external ID")
-        (is (= id (:id entity))
-            "The submitted entity is linked to the external ID")))
+        (when (is (= 200 (:status response)))
```

Comment:
can we have other response code in this test? It seems that checking if the response is a 200 should be in an assert to validate that the test context is properly initialized.